pkgs/tools/misc/pandoc-include/default.nix

```
@@ -3,34 +3,35 @@
 , lib
 , natsort
 , panflute
+, lxml
 , setuptools
 }:
 
 buildPythonApplication rec {
   pname = "pandoc-include";
-  version = "1.2.1";
+  version = "1.3.0";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "DCsunset";
     repo = pname;
```

```suggestion
    repo = "pandoc-include";
```
https://github.com/NixOS/nixpkgs/issues/277994